### ISSUE TITLE
CAF enhancement: Add 'TIKL to approve SNAP without SHEL in 10 days checkbox

### ISSUE THREAD

**Issue by IlseFerris** [MN-Script-Team/DHS-MAXIS-Scripts#2911]
This is an area of high error in SNAP. If SNAP is not expedited, workers are supposed to give clients Add 'TIKL to approve SNAP without SHEL in 10 days' checkbox and associated TIKL message.

**Comment by C-Love** [comment]
I have questions about this. We discuss this a LOT and there are kind of 2 schools of thought. 

1. Case should be approved if all mandatory verifications are received - without the SHEL included if necessary. Then supplements should be issued if the SHEL verif is rec'vd in the processing period. This way there is no delay to benefits for the client. 

2. No SNAP should be issued until 10 days have been allowed to let client provide the SHEL verif and have the benefits issued. This tends to be easier for the workers - only need to approve cases once, and no chance of having to issue a MONY CHCK for a supplement BUT client's SNAP is delayed. This also raises the question of if the supplement should be issued if the verification is received after 10 days but within the processing period. 

Mostly I think the wording is what we need to determine on this one and which script it is best attached to. I can see if SNAP is being approved without the deduction then the checkbox would be better on approved programs.

**Comment by Farleight** [comment]
Option 1 doesn't really mesh with policy as I understand it... I mean, that's absolutely how it SHOULD be handled... and very possibly how it is handled even by people who know better... but I don't know that we can point the script in that direction.

**Comment by C-Love** [comment]
Policy states to determine eligibility as soon as the necessary information is available and only lists mandatory verifications and inconsistent information before the approval. We have been told quite firmly that SHEL is not mandatory for eligibility. This is definitely something DHS could weigh in on @ruthmaloney  - maybe you have some information. We have discussed this in some length here and would love some input.

**Comment by Farleight** [comment]
DHS Trainers (in the SNAP refresher course last month) pretty much told us outright that option 1 was wrong. I admittedly haven't looked it up since then because I would hate to prove them right. If there are differing DHS opinions on it, I'll happily listen to them.

**Comment by C-Love** [comment]
I didn't get to go that special refresher but the teammate of mine that went said it was fairly case management focused. That is my understanding for case management when dealing with ongoing cases. My lens on this is certainly from an intake perspective. Basically we have been told to NOT delay benefits to clients for non-mandatory reasons and that we are very often for SNAP over verifying. 

Now, I could be completely wrong about this but it seems this may differ based on where the client is at in their case process.

**Comment by Farleight** [comment]
No, it definitely differs. They were specifically addressing the initial intake at the time though. And this is all hearsay at this point. I could be completely misinterpreting what they said too. I hope that I was, because Option 1 just makes sense.

**Comment by ruthmaloney** [comment]
It’s an excellent question that we’ve been waiting for an answer to since June 2015. The question is currently at the national FNS level. The last update we received was in March, and that was to tell us that they (FNS) still have not decided on an answer due to multiple other policy areas that get involved.

Currently there are Policy Quest answers and different parts of the Combined Manual that support both ways of processing (waiting ten days or moving forward without allowing the deduction). I’ve heard the suggestion that until we get an answer, workers should ask the client what they would prefer and case note the client’s choice prior to proceeding. It needs to be clear that this method should only be used for non-mandatory verifications (such as shelter) and the worker should still send/give the client a 2919 request form whether the client chooses to proceed or wait.

I’m sorry I can’t provide a more definitive answer to the question. Until we have an answer, I like the idea of giving (and noting) the options to the client and letting them make the decision.

SNAP is so easy right?  ;-)

Ruth Maloney

**Comment by C-Love** [comment]
That is really helpful input. We get an 'Everyone is Right for Now' award. Thank you @ruthmaloney

**Comment by IlseFerris** [comment]
Thanks for all the input team! @ruthmaloney this is very really good to know.

**Comment by IlseFerris** [comment]
@ruthmaloney In 10/17, our ME reviewer presented a technical assistance training and indicated that SNAP can be approved without giving the client 10 days ONLY IF the client is aware that a supplement cannot be issued and that their benefit may be reduced (and case noted thoroughly.) Can you confirm this/point us in the direction of documentation regarding this policy clarification? Thank you!

**Comment by ruthmaloney** [comment]
That information is correct, we received clarification from FNS in July 2017. The following PQ’s have been answered addressing this situation: 16055, 16169, 17822, 15519, 16979, 15909, 16069, 16489, 16716, 16769.

**Comment by C-Love** [comment]
Great @ruthmaloney - good to know. 

So we will need to create functionality that the worker will need to either address that there will be a TIKL for 10 days out if NOT approving without the shelter deduction OR an approval will be done without shelter expense budgeted AND will very specifically require the worker to indicate they explained that to the client. 

This is also functionality that should be addressed in Eligibility Summary. I will add that to the issue.
